files/scripts/sonic-dpu-mgmt-traffic.sh

```diff
@@ -184,7 +184,7 @@ ctrl_dpu_ib_forwarding(){
         dpu_name="${sel_dpu_names[$index]}"
         dpu_midplane_ip="${midplane_ip_dict[$dpu_name]}"
         switch_port="${provided_ports[$index]}"
-        add_rem_valid_iptable $op nat POSTROUTING -p tcp -d $dpu_midplane_ip --dport $dest_port -j MASQUERADE
+        add_rem_valid_iptable $op nat POSTROUTING -p tcp -d $dpu_midplane_ip --dport $dest_port -j SNAT --to-source $midplane_gateway
```

Comment:
@yue-fred-gao If midplane-gateway IP gets updated through yang or config, does that flow trigger this script to change the source-ip?